src/components/form-elements/Input.jsx

```diff
@@ -38,10 +70,10 @@ export const COLOR = Object.freeze({
 });
 
 type PropsType = {
-  type?: $Values<typeof TYPE>,
+  type?: InputType,
```

Comment:
props with unions, such as `InputType`, `InputSizeType`, `InputColorType` is better to make nullable: `type?: ?InputType`, because in main app we use patterns like
`color={ ifSomething ? COLOR : null}` and such patterns will be broken

Reply:
@PinkaminaDianePie ah, yeah. sure, I will change that in a sec. thx for feedback. 🙂